examples/vector-tiles-4326.js

```diff
@@ -1,55 +1,36 @@
-import MVT from '../src/ol/format/MVT.js';
 import TileGrid from '../src/ol/tilegrid/TileGrid.js';
-import VectorTileSource from '../src/ol/source/VectorTile.js';
-import View from '../src/ol/View.js';
+import {Map, View} from '../src/ol/index.js';
 
```

Comment:
Curious if there is a reason for this blank line (like forcing a certain import order).  If so, a comment would clarify.

Reply:
This was just an oversight on my end, I removed the line break while including @mike-000's suggested simplification.